roles/acm_attach_spoke_cluster/README.md

```diff
@@ -0,0 +1,30 @@
+ACM Attach Spoke Cluster
+=========
+
+This role allows to attach a spoke cluster to a given hub cluster.
+
+This is based on the following [OCP documentation](https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.5/html/clusters/managing-your-clusters#importing-a-target-managed-cluster-to-the-hub-cluster).
+
+Three resources are created in this role:
+
+- ManagedCluster resource (and related Namespace is automatically created after its creation).
+- Autoimport Secret.
+- KlusterletAddonConfig.
+
+Requirements
+------------
+
+* Active spoke cluster already created, having access to its kubeconfig.
+
+Role Variables
+--------------
+
+Name                         | Type   | Required | Default                                            | Description
+---------------------------- | ------ | -------- | -------------------------------------------------- | ------------------------------------------------------------
+aasc_cluster_kubeconfig_path | string | yes      | -                                                  | Path to spoke cluster's kubeconfig file
+aasc_cluster_name            | string | yes      | -                                                  | Cluster name, used for identifying the ManagedCluster and the Namespace
+
+License
+-------
+
+GNU GENERAL PUBLIC LICENSE version 3
```

Comment:
Why GPLv3?

Reply:
I copied the role template from this another one: https://github.com/redhatci/ansible-collection-redhatci-ocp/tree/main/roles/configure_ztp_gitops_apps#license, so if this needs to be fixed, we need to check other roles too.

I'll remove from here for the moment.